content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -227,3 +250,153 @@ use our connector as an alternative payroll solution.
 .. important::
    To :ref:`configure the Employment Hero API <employment_hero/configuration>` for **United
    Kingdom**, use the following value as :guilabel:`Payroll URL`: `https://api.yourpayroll.co.uk/`.
+
+.. _localization/united-kingdom/cis-deduction:
+
+.. |HMRC| replace:: :abbr:`HMRC (HM Revenue and Customs)`
+.. |CIS| replace:: :abbr:`CIS (Construction Industry Scheme)`
+
+CIS deduction
+=============
+
+The Construction Industry Scheme deduction (CIS deduction) is a tax deduction system used in the UK
+designed specifically for the construction industry. It requires contractors to deduct a percentage
+of payments made to subcontractors and forward these deductions to HM Revenue and Customs (HMRC).
+These deductions apply only to the labor portion of the payments and serve as advance payments
+towards the subcontractor's tax and National Insurance contributions. Contractors are required to
+register for the scheme, but subcontractors are not. However, subcontractors who are not registered
+face higher payment deductions. Under the |CIS|, contractors must deduct 20% from payments to
+registered subcontractors, while the deduction increases to 30% for unregistered ones.
+
+.. seealso::
+
+   - `Construction Industry Scheme (CIS) <https://www.gov.uk/what-is-the-construction-industry-scheme>`_
+   - `Guidelines for CIS contractors
+     <https://www.gov.uk/what-you-must-do-as-a-cis-contractor>`_
+   - `Guidelines for CIS subcontractors
+     <https://www.gov.uk/what-you-must-do-as-a-cis-subcontractor>`_
+
+As a contractor, you are required to register with the |CIS| before hiring subcontractors and to
+check whether each subcontractor is registered with the |CIS|. You must also maintain records of all
+payments and deductions and submit monthly returns to HMRC, including the following details:
+
```

Comment:
nitpicking but since the list items are not full sentence, i would remove the periods?